pkgs/games/steam/fhsenv.nix

```diff
@@ -234,6 +235,12 @@ in buildFHSEnv rec {
     #
     # [1] <https://github.com/libsdl-org/SDL/commit/8e2746cfb6e1f1a1da5088241a1440fd2535e321>
     export SDL_JOYSTICK_DISABLE_UDEV=1
+
+    # Following XDG spec [1], XDG_DATA_DIRS should default to "/usr/local/share:/usr/share".
+    # In nix, it is commonly set without containing these values, so we add them as fallback.
+    #
+    # [1] <https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html>
+    export XDG_DATA_DIRS=$XDG_DATA_DIRS''${XDG_DATA_DIRS:+:}/usr/local/share:/usr/share
```

Comment:
[BuildFHSEnv](https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/build-fhsenv-bubblewrap/buildFHSEnv.nix#L76-L77) extending `XDG_DATA_DIRS` and [fhs-environments docs](https://github.com/NixOS/nixpkgs/blob/master/doc/builders/special/fhs-environments.section.md) on the [last line](https://github.com/NixOS/nixpkgs/blob/master/doc/builders/special/fhs-environments.section.md?plain=1#L49) suggesting it handles `gsettings` might mean the right place to have this is in the fhsenv code instead.

Reply:
Agreed, I was a little hesitant to modify `BuildFHSEnv` since this is firstly a bugfix that will need to backported.

Reply:
We can send back a different fix if the change is intrusive enough, and the fix more correct.